cli/src/main.rs

```diff
@@ -86,7 +99,7 @@ fn main() {
                 .long("use")
                 .takes_value(true)
                 .possible_values(&["cfg", "disassembler", "interpreter", "jit"])
-                .required(true),
+                .required_unless_present("debug"),
```

Comment:
How about `--use debugger` instead?

Reply:
sounds good, plus an extra option to set the `port`? Or just hardcode it to `9001`?

Reply:
Option for the port (with a default setting) sounds good.